lib/restaurant.rb

```diff
@@ -0,0 +1,19 @@
+require_relative 'takeaway'
+
+class Restaurant < TakeAway
```

Comment:
This class is superfluous as it adds no additional behaviour.  Just use `Takeaway`